### DISCUSSION TITLE
Compatibility repository

### DISCUSSION THREAD

**Discussion by SinUsername** [skyline-emu/skyline#1462] [General]
Hi I'm looking for the compatibility repository Since I want to know which games are working and which are not and also if I can also contribute in the event that I discover one in the case. I can not enter the discord because I lost my xD accounts in short help me

**Comment by hacobotdev** [comment]
There you go https://github.com/skyline-emu/skyline-games-list/issues

**Reply by SinUsername** [reply]
Thanks
